crates/uv-configuration/src/dev.rs

```diff
@@ -125,36 +125,34 @@ impl GroupsSpecification {
                 [group] => Some(Cow::Owned(format!("--only-group {group}"))),
                 [..] => Some(Cow::Borrowed("--only-group")),
             },
-            Self::Explicit { include } => match include.as_slice() {
-                [] => Some(Cow::Borrowed("--no-default-groups")),
-                [group] => Some(Cow::Owned(format!("--group {group}"))),
-                [..] => Some(Cow::Borrowed("--group")),
-            },
         }
     }
 
     /// Iterate over all groups referenced in the [`GroupsSpecification`].
     pub fn names(&self) -> impl Iterator<Item = &GroupName> {
         match self {
-            GroupsSpecification::Include { include, exclude } => {
-                Either::Left(include.names().chain(exclude.iter()))
-            }
-            GroupsSpecification::Only { include, exclude } => {
-                Either::Left(include.iter().chain(exclude.iter()))
-            }
-            GroupsSpecification::Explicit { include } => Either::Right(include.iter()),
+            GroupsSpecification::Include {
+                include, exclude, ..
+            } => include.names().chain(exclude.iter()),
+            GroupsSpecification::Only { include, exclude } => include.iter().chain(exclude.iter()),
         }
     }
 
     /// Returns `true` if the specification includes the given group.
-    pub fn contains(&self, group: &GroupName) -> bool {
+    pub fn contains(&self, group: &GroupName, is_default_group: bool) -> bool {
```

Comment:
The `is_default_group: bool` argument is not ideal, but it seems to be the simplest solution I could come up with.